pom.xml

```diff
@@ -70,6 +70,10 @@
     <!-- <module>drools-examples-android</module> -->
   </modules>
 
+  <properties>
+    <findbugs.failOnViolation>true</findbugs.failOnViolation>
+  </properties>
+
```

Comment:
@rsynek could you please rebase this with the current master? Recent change introduced a `<properties>` tag as this is duplicating it (which maven does not like). See https://github.com/droolsjbpm/droolsjbpm-integration/blob/master/pom.xml#L23